Introduce issue templates

raphaelrobert opened the pull request:
This PR introduces issue templates. This is working towards #196.

kkohbrok commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -0,0 +1,30 @@
+1. Use the following API call/function: ...
+2. See error
+
+**Expected behavior**
```
In addition to "Expected Behaviour", maybe have a field called "Actual Behaviour"?

franziskuskiefer commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -0,0 +1,33 @@
+If applicable, add screenshots or debug logs to help explain your problem.
+
+**Platform (please complete the following information):**
+ - OS: [e.g. macOS]
```
OS with version would be even better.

franziskuskiefer reviewed (commented):
Thanks! A couple thought, no strong opinions.

Maybe give all of them a header with something like 
"Issue tracker is **ONLY** used for reporting bugs, feature requests, and spec changes. For support or discussions please use https://github.com/openmls/openmls/discussions (or some chat service)"
Also, I'm not sure we want feature requests in here. They are usually better handled in discussions before filing an issue.

There are different approaches to how to put the text in. I like when they are commented out (`<!--- -->`) because they can stay in when writing the ticket.

raphaelrobert commented:
I addressed your remarks.
I left feature requests in, because not everything fits into the bug/spec change categories. We can try this for ourselves and change this later if we see it doesn't work.
